### PR TITLE
add acc_basic setup

### PR DESCRIPTION
**Pull request by nutrik**

### CODE REVIEW

File: veros/setup/acc_basic/acc_basic.py [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+        vs.eke_crhin = 1.0
+        vs.eke_lmin = 100.0
+        vs.enable_eke_superbee_advection = False
+        vs.enable_eke_isopycnal_diffusion = False
+
+        vs.enable_idemix = False
+        vs.enable_idemix_hor_diffusion = False
+        vs.enable_eke_diss_surfbot = False
+        vs.eke_diss_surfbot_frac = 0.2
+        vs.enable_idemix_superbee_advection = False
```
Comment (dionhaefner) [inline veros/setup/acc_basic/acc_basic.py:81]: Let's just remove all the stuff we're not using, this is supposed to be basic after all :)

File: veros/setup/acc_basic/acc_basic.py [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+        vs._t_star[vs.yt > 20] = 15 * (1 - (vs.yt[vs.yt > 20] - 20) / (yt_max - 20))
+        vs._t_rest = vs.dzt[None, -1] / (30. * 86400.) * vs.maskT[:, :, -1]
+
+        if vs.enable_tke:
+            vs.forc_tke_surface[2:-2, 2:-2] = np.sqrt((0.5 * (vs.surface_taux[2:-2, 2:-2] + vs.surface_taux[1:-3, 2:-2]) / vs.rho_0)**2
+                                                      + (0.5 * (vs.surface_tauy[2:-2, 2:-2] + vs.surface_tauy[2:-2, 1:-3]) / vs.rho_0)**2)**(1.5)
+
+        if vs.enable_idemix:
+            vs.forc_iw_bottom[...] = 1e-6 * vs.maskW[:, :, -1]
+            vs.forc_iw_surface[...] = 1e-7 * vs.maskW[:, :, -1]
```
Comment (dionhaefner) [inline veros/setup/acc_basic/acc_basic.py:133]: Same here

File: veros/setup/acc_basic/acc_basic.py [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+
+    @veros_method
+    def set_forcing(self, vs):
+        vs.forc_temp_surface[...] = vs._t_rest * (vs._t_star - vs.temp[:, :, -1, vs.tau])
+
+    @veros_method
+    def set_diagnostics(self, vs):
```
Comment (dionhaefner) [inline veros/setup/acc_basic/acc_basic.py:140]: I think the spec here was "no snapshots"

Comment (nutrik) [reply veros/setup/acc_basic/acc_basic.py:140]: Exactly! So I removed snapshots + Markus wanted me to remove energy output as well.
Do you want me to formalise their absence somehow?

Comment (dionhaefner) [reply veros/setup/acc_basic/acc_basic.py:140]: Oh, you did? I thought snapshots are active by default, but maybe I remember it wrong.

Comment (nutrik) [reply veros/setup/acc_basic/acc_basic.py:140]: I ran the model for a test and neither snapshots nor energy output files were created.

**Pull request merged**